_minutes/2021-09-02-wecg.md

```diff
@@ -0,0 +1,109 @@
+
+* [timothy] what to do with unknown properties. Know that Chrome is somewhat strict and Safari/Firefox more flexible. Given the cross-browser nature we should document the desired behavior of unknown properties, ideally to not treat them as severe errors.
+* [tomislav] The common behavior is that implementations throw when known keys have invalid types.
+* [glazou] don’t forget original context ; extension vendors don’t want to have to generate multiple manifests on a per-browser basis. Ideally unsupported properties should be ignored so that one manifest can be used in multiple browsers.
+* [timothy] Safari generalizes chrome\_ to browser\_ (e.g. supporting browser\_settings\_overrides in addition to chrome\_settings\_overrides). I would imagine Chrome to be throwing if it detected browser\_settings\_overrides.
+* https://github.com/w3c/webextensions/issues/14#issuecomment-894327417
+  * [rob] Someone asks for support to have the ability to override top-level properties in a subkey (e.g. browser\_specific\_settings).
+  * [glazou] Would ideally not see a new reality where browsers fake their browsers to accommodate extensions.
+  * [rob] At some point in the past I wanted the ability to override manifest keys for specific browser versions to avoid errors in browsers that did not recognize the new value. This benefit does not weigh against the complexity of enforcing and validating the manifest though, e.g. validators unaware of this override could inadvertently be bypassed.
+  * [timothy] Safari’s only critical errors are if JSON is incorrect or the manifest version is invalid.
+  * [simeon] will file crbug
+
+Manifest content, follow-up to previous week ([issue 60](https://github.com/w3c/webextensions/issues/60)).
+
+* [simeon] Should the manifest account for things like maximum name length imposed by web stores?
+* [glazou] Put constraints in an informal note if it’s not strictly part of the spec.
+* [simeon] We could put the maximum length in the JSON schema, so that external tooling can validate the schemas.
+* [timothy] Against constraints since different platforms may have different needs for the length, e.g. on mobile there may be even less space.
+* [simeon] Chrome doesn’t enforce limitations on short\_name, the Chrome Web Store does.
+* [tomislav] While I’m in favor of adding informal notes, note that store policies are explicitly outside the scope of this group.
+* [simeon] Consensus is to add a non-normative note.
+* [timothy] It is a good idea to call out potential issues in the spec, even if informally.
+
+Is the extension security model in scope for the spec? (Jonathan Kingston)
+
+* [jkt] Is it possible to describe the security model in the spec, for example the practice where browsers maintain a set of restricted URLs that are shielded from extensions.
+* [rob] Please file a new issue in the repo so that we can discuss this in more detail at the next meeting.
```

Comment:
```suggestion
* [rob] Please file a new issue in the repo so that we can discuss this in more detail at the next meeting.
* jkt opened [issue 71](https://github.com/w3c/webextensions/issues/71) after the meeting.
```